google-http-client/src/test/java/com/google/api/client/util/SecurityUtilsTest.java

```diff
@@ -160,4 +166,51 @@ public void testVerifyX509() throws Exception {
   public void testVerifyX509WrongCa() throws Exception {
     assertNull(verifyX509(TestCertificates.BOGUS_CA_CERT));
   }
+
+  public void testCreateMtlsKeyStoreNoCert() throws Exception {
+    final InputStream certMissing =
+        getClass()
+            .getClassLoader()
+            .getResourceAsStream("com/google/api/client/util/privateKey.pem");
+    IllegalArgumentException exception =
+        assertThrows(
+            IllegalArgumentException.class,
+            new ThrowingRunnable() {
+              @Override
+              public void run() throws Throwable {
+                SecurityUtils.createMtlsKeyStore(certMissing);
+              }
+            });
+    assertThat(exception)
+        .hasMessageThat()
+        .isEqualTo("certificate is missing from certAndKey string");
+  }
+
+  public void testCreateMtlsKeyStoreNoPrivateKey() throws Exception {
+    final InputStream privateKeyMissing =
+        getClass().getClassLoader().getResourceAsStream("com/google/api/client/util/cert.pem");
+    IllegalArgumentException exception =
+        assertThrows(
+            IllegalArgumentException.class,
+            new ThrowingRunnable() {
+              @Override
+              public void run() throws Throwable {
+                SecurityUtils.createMtlsKeyStore(privateKeyMissing);
+              }
+            });
+    assertThat(exception)
+        .hasMessageThat()
+        .isEqualTo("private key is missing from certAndKey string");
+  }
+
+  public void testCreateMtlsKeyStoreSuccess() throws Exception {
+    InputStream certAndKey =
+        getClass()
+            .getClassLoader()
+            .getResourceAsStream("com/google/api/client/util/mtlsCertAndKey.pem");
+
+    KeyStore mtlsKeyStore = SecurityUtils.createMtlsKeyStore(certAndKey);
+
+    assertEquals(mtlsKeyStore.size(), 1);
```

Comment:
In `assertEquals`, the expected value comes first.

```suggestion
    assertEquals(1, mtlsKeyStore.size());
```